falcon/api.py

```diff
@@ -186,6 +187,7 @@ def __init__(self, media_type=DEFAULT_MEDIA_TYPE,
         self.resp_options.default_media_type = media_type
 
         # NOTE(kgriffs): Add default error handlers
+        self.add_error_handler(Exception, self._python_error_handler)
```

Comment:
I'm wondering if this should be `Exception` or `BaseException`... @kgriffs thoughts?

Reply:
I would suggest just _Exception_. The base one could make the WSGI app catch stuff like system exit, keyboard interrupt / Ctrl+C etc, and return JSON in those cases. That doesn't sound like safe default behaviour, but maybe we document how to, in case anyone still wants that thing.

Reply:
So, are we okay with this as-is?

Reply:
Yeah, we're good with `Exception`, my bad.